__tests__/components/Button.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Button } from '@/components/Button';
+Button;
+
+describe('Button', () => {
+    it('should render', () => {
```

Comment:
please correct the `it` statement, "should render" what??

Reply:
done.